esmvaltool/cmorizers/obs/cmorize_obs_woa.py

```diff
@@ -4,67 +4,127 @@
    Tier 2: other freely-available dataset.
 
 Source
-   https://data.nodc.noaa.gov/woa/WOA13/DATAv2/
+   WOA18: https://www.ncei.noaa.gov/access/world-ocean-atlas-2018/
 
 Last access
-   20190131
+   WOA18: 20210311
 
 Download and processing instructions
-   Download the following files:
-     temperature/netcdf/decav81B0/1.00/woa13_decav81B0_t00_01.nc
-     salinity/netcdf/decav81B0/1.00/woa13_decav81B0_s00_01.nc
-     oxygen/netcdf/all/1.00/woa13_all_o00_01.nc
-     nitrate/netcdf/all/1.00/woa13_all_n00_01.nc
-     phosphate/netcdf/all/1.00/woa13_all_p00_01.nc
-     silicate/netcdf/all/1.00/woa13_all_i00_01.nc
+   All handled by the script (download only if local data are missing)
```

Comment:
It could be good to keep the list of files to download. It is nicer to have an automated download procedure but the cmorizer may also be run on machines without internet access for which WOA data would need to be downloaded manually.

Reply:
> * Some of these issues also appear in the "old cmorizer, 

The cmorizer uses the function [`fix_var_metadata`](https://github.com/ESMValGroup/ESMValTool/blob/615b935719c8dfbd328b3c5769c13f82228ff94d/esmvaltool/cmorizers/obs/utilities.py#L213) to set the cube `name`, `long_name`, `units` with values from the CMOR tables.

I guess it may relate to the fact that it is based on CMIP5 tables ...

Reply:
I think the list of files will become very long as it has to be done for both datasets and the commented preamble will be far too long ...

Reply:
I added the list of files and comment in 4512652.